src/Template/Element/login.ctp

```diff
@@ -35,7 +35,7 @@ $registerUrl = $this->Url->build([
 ]);
 ?>
 
-<div ng-controller="LoginDialogController as vm" layout="row" layout-align="center center" flex>
+<div ng-controller="LoginDialogController as vm" layout="row" layout-align="center center" flex hide-xs hide-sm>
```

Comment:
`hide-xs` displays when width < 600px.
`hide-sm` displays when 600px <= width < 960px.
I think their combination can be replaced by `show-gt-sm`, which displays when width >= 960px.

Reply:
The `show` directive is only negating `hide`. So `show-gt-sm` alone will still show the element even if the width is in the `xs` or `sm` range. To show *only* when width >= 960px, it would have to be `hide show-gt-sm`.